src/msg/simple/Pipe.cc

```diff
@@ -2125,16 +2200,27 @@ int Pipe::do_sendmsg(struct msghdr *msg, int len, bool more)
       assert(l == len);
     }
 
-    int r = ::sendmsg(sd, msg, MSG_NOSIGNAL | (more ? MSG_MORE : 0));
+    int r;
+#if defined(MSG_NOSIGNAL)
+    r = ::sendmsg(sd, msg, MSG_NOSIGNAL | (more ? MSG_MORE : 0));
```

Comment:
3b34e2f36358bd93a16d38cbf9e5a464d138a79b

Hrm, hard to say what that was breaking before.  Maybe it's not a problem..